services/blockrelay/standard/auctionblock.go

```diff
@@ -103,17 +104,33 @@ func (s *Service) cacheBid(_ context.Context,
 	bid *builderspec.VersionedSignedBuilderBid,
 ) {
 	if bid == nil {
+		epoch := s.chainTime.SlotToEpoch(slot)
+
 		// No bid supplied; create a dummy for the purposes of caching so that if asked for this bid
 		// we can actively respond to say we don't have anything (as opposed to attempting to fetch
 		// a bid when we're queried).
 		s.log.Trace().Msg("Bid is nil; creating dummy")
-		bid = &builderspec.VersionedSignedBuilderBid{
-			Version: spec.DataVersionDeneb,
-			Deneb: &deneb.SignedBuilderBid{
-				Message: &deneb.BuilderBid{
-					Value: uint256.NewInt(0),
+		// It would be nice if we could avoid doing this as it would remove the need to set the electraForkEpoch on the service.
```

Comment:
Do we care about the fork version of this bid?  From what I remember (and this is not looking at the code right now) we only use this dummy bid to call `Value()`, so we can get away with always returning with the deneb version specified and populated, with no ill effects.